pages/common/mpv.md

```diff
@@ -9,3 +9,7 @@
 - Play a video at specified speed (0.01 to 100, default 1):
 
 `mpv --speed {{speed}} {{video_file}}`
+
+- Play a video using a profile defined in your `mpv.conf` file:
```

Comment:
How about we **drop** the `your`.
'Play a video using a profile defined in `mpv.conf` file:'

Reply:
I don't really agree, because the user has to define his own `mpv.conf` file. I could write 'in _the_ `mpv.conf`' file, but that would make it sound like it already exists.

Reply:
I think this is fine as-is here, as the `mpv.conf` file is likely to be in the user's _own_ home directory.

Reply:
I agree

Reply:
FYI - We try to avoid personal pronouns and aim to keep the language impersonal. If you want to keep `your`, I would suggest `one's` or something similar which makes it sound impersonal.

Reply:
I've pushed a new one using `the`, I think it reads better than `one's`.